annotations/management/commands/import_tenses.py

```diff
@@ -30,59 +30,78 @@ def handle(self, *args, **options):
 
 
 def process_file(f, language, use_other_label, model='annotation'):
```

Comment:
`use_other_label` is not used anymore (as all labels are checked), so this can go (also from the view and the command line expression)